configserver/src/main/java/com/yahoo/vespa/config/server/ConfigServerBootstrap.java

```diff
@@ -37,8 +37,10 @@
  * Main component that bootstraps and starts config server threads.
  *
  * If config server has been upgraded to a new version since the last time it was running it will redeploy all
- * applications. If that is done successfully the RPC server will start and the health status code will change from
- * 'initializing' to 'up' and the config server will be put into rotation (start serving status.html with 200 OK)
+ * applications. If that is done successfully the RPC server the health status code will change from
```

Comment:
"RPC server the health status" -> " RPC server will start and the health status" ?

Reply:
Yes, will fix